chia/full_node/hint_store.py

```diff
@@ -38,3 +38,13 @@ async def add_hints(self, coin_hint_list: List[Tuple[bytes32, bytes]]) -> None:
             [(None,) + record for record in coin_hint_list],
         )
         await cursor.close()
+
+    async def count_hints(self) -> int:
+        async with self.coin_record_db.execute("select count(*) from hints") as cursor:
+            row = await cursor.fetchone()
+
+        if row is None:
```

Comment:
Is this none with an empty table? If not, we should use:
`assert row is not None`

Reply:
Also needs a test

Reply:
TBH I'm not sure when this is none - if the table is empty, the returned value is just `0` for the count, not `None` here - I think this is just a possible result from the general `execute` call on the DB - if None happens here, something is messed up with the DB.